store/gcworker/gc_worker.go

```diff
@@ -2183,3 +2183,16 @@ func (s *mergeLockScanner) physicalScanLocksForStore(ctx context.Context, safePo
 
 	return nil
 }
+
+// GCHandler runs garbage collection job.
+type GCHandler interface {
+	// Start starts the GCHandler.
+	Start()
+
+	// Close closes the GCHandler.
+	Close()
+}
+
+// NewGCHandlerFunc creates a new GCHandler.
+// To enable real GC, we should assign the function to `gcworker.NewGCWorker`.
+var NewGCHandlerFunc func(storage tikv.Storage, pdClient pd.Client) (GCHandler, error)
```

Comment:
I think we can remove it and use `NewGCWorker` directly.

Reply:
Let me have a try